schemas/access/role-1.yml

```diff
@@ -173,7 +173,19 @@ properties:
                   - /access/bot-1.yml
                   - /dependencies/dynatrace-environment-1.yml
   ldapGroup:
-    "$ref": "/common-1.json#/definitions/ldapGroupName"
+    type: object
+    properties:
+      name:
+        "$ref": "/common-1.json#/definitions/ldapGroupName"
+        description: The name of the LDAP group
+      notes:
+        type: string
+        description: Notes added to the LDAP group
```

Comment:
nit: Should we add here that this is targeted for Rover groups?

Reply:
Mentioned rover in the last commit too